src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -31,35 +31,248 @@
  */
 
 #include "vr_cooperation/vr_module.h"
+#include "vr_cooperation/event_engine/event_dispatcher.h"
+#include "json/json.h"
```

Comment:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
It is follow Google coding style, the order of includes: Related header, C library, C++ library, other libraries' .h, your project's .h.
vr_module.cc is implement of vr_module.h -> vr_module.h must be included first.